index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
         <img src="#" alt="FontIcon" class="font-icon" />
         <img src="#" alt="OtherImg" class="fourth-illustration" />
       </figure>
-      <section class="product-container">
-        <ul class="product-list">
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
+    </main>
+    <section class="product-container">
+      <ul class="product-list">
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
```

Comment:
it's a links to pages with this category 
and you need to add text, not only image